b2sdk/transfer/inbound/downloaded_file.py

```diff
@@ -80,6 +80,11 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 
 class DownloadedFile:
+    """
+    Result of a successful download initialization. Holds information about file's metadata
+    and allows to perform the download, determining the download strategy.
```

Comment:
I'd remove `, determining the download strategy.` because if we one day will change it so that setting strategy is automatic, we'll have to upgrade apiver to remove that part of the docstring. This choosing of strategy is not a fundamentally important property of the class, so lets not mention it as everything that you mention here becomes part of the public interface.